maintainers/maintainer-list.nix

```diff
@@ -10106,4 +10106,9 @@
     github = "zupo";
     githubId = 311580;
   };
+  ishan = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add ishan`` and please sort it alphabetically.